demozoo/templates/base.html

```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
-<html class="no-js" lang="en">
```

Comment:
It's becoming increasingly uncommon to use modernizr tests to style against. Besides, we never used it in all those years 🥇